tkp/db/database.py

```diff
@@ -91,11 +90,13 @@ def __init__(self, **kwargs):
                                              self.host,
                                              self.port,
                                              self.database),
-                                            echo=False
+                                            echo=False,
+                                            poolclass=NullPool,
                                             )
         self.Session = sessionmaker(bind=self.alchemy_engine)
```

Comment:
But it is not a factory right?

https://en.wikipedia.org/wiki/Factory_method_pattern